grammars/prql-lezer/src/prql.grammar

```diff
@@ -58,7 +58,8 @@ Op_bin { op_bin_only | !term op_unary }
   // TODO: does excluding spaces here work? It seems to make a difference, but
   // then I think the `Number` rule doesn't allow spaces...
   Date { "@" ![ ] (( @digit | "-" ) ![ ] )+ }
-  ident_part { @asciiLetter (@asciiLetter | "_" | @digit )* }
+  identifier_char { @asciiLetter | $[_\u{a1}-\u{10ffff}] }
+  ident_part { identifier_char (@asciiLetter | "_" | @digit )* }
```

Comment:
I don't think this is quite right — this means the unicode character can only be at the start of the `ident_part`?

I do think tests would be very very helpful here. There's a nice example in https://lezer.codemirror.net/examples/test/ if you'd be up for doing that separately...

Reply:
You're not, it's not quite right, and I had a feeling it wasn't but it worked, but now I see that it produces multiple `Expr`.